indigo/indigo/src/main/scala/indigo/shared/datatypes/RGB.scala

```diff
@@ -51,6 +51,24 @@ object RGB:
   val White: RGB   = RGB(1, 1, 1)
   val Black: RGB   = RGB(0, 0, 0)
 
+  val Coral: RGB     = RGBA.Coral.toRGB
```

Comment:
@davesmith00000 I considered writing a function to convert instances of `RGBA` to `RGB` implicitly to make this section more concise but wasn't sure what casing would be best. `rgbaToRGB`?

Reply:
Explicit is totally fine, preferable even.